deepchem/models/torch_models/dtnn.py

```diff
@@ -0,0 +1,262 @@
+import numpy as np
+from typing import List
+
+import torch.nn as nn
+import torch
+
+from deepchem.models.losses import L2Loss
+from deepchem.models.torch_models import layers
+from deepchem.models.torch_models import TorchModel
+
+
+class DTNN(nn.Module):
+    """Deep Tensor Neural Networks
+
+    In this class, we establish a sequential model for the Deep Tensor Neural Network (DTNN) [1]_.
+
+    References
+    ----------
+    .. [1] Schütt, Kristof T., et al. "Quantum-chemical insights from deep
+        tensor neural networks." Nature communications 8.1 (2017): 1-8.
+
+    """
+
+    def __init__(self,
+                 n_tasks,
+                 n_embedding=30,
+                 n_hidden=100,
+                 n_distance=100,
+                 distance_min=-1,
+                 distance_max=18,
+                 output_activation=True,
+                 mode="regression",
+                 dropout=0.0,
+                 **kwargs):
+        """
+        Parameters
+        ----------
+        n_tasks: int
```

Comment:
mention the default value of the arguments in the comments too

Reply:
Done